dev/com.ibm.websphere.appserver.features/visibility/public/cdi-2.0/com.ibm.websphere.appserver.cdi-2.0.feature

```diff
@@ -62,7 +62,8 @@ Subsystem-Name: Contexts and Dependency Injection 2.0
  com.ibm.ws.cdi.2.0.weld, \
  com.ibm.websphere.javaee.jaxb.2.2; apiJar=false; require-java:="9"; location:="dev/api/spec/,lib/",\
  com.ibm.websphere.javaee.jaxws.2.2; apiJar=false; require-java:="9"; location:="dev/api/spec/,lib/", \
- com.ibm.ws.cdi.interfaces
+ com.ibm.ws.cdi.interfaces, \
+ com.ibm.websphere.appserver.spi.cdi
```

Comment:
I think the spi "bundle" needs to be listed in the jars section just below
-jars=com.ibm.websphere.appserver.spi.cdi; location:=dev/spi/ibm/

Reply:
and not here because osgi wise, the package is already being included via the cdi.interfaces bundle